snippets/accessibility/AccessibleLinks.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Snippet Title: Accessible Link Colors
+ * Scope:         global
+ * Goal:          accessibility
+ * Description:   Makes most links across the site blue and underlined, and purple if visited. Includes activity links, but excludes section titles, buttons and other link controls across the site
+ * Creator:       Pierre Camaly de Brosses
+ * Tracker issue: <snippet tracker issue>
+ * Tested on:     Moodle 4.5, Firefox, Chrome and Safari
+ * Usage note:    None
+ *
+ * @copyright     2025 University of Exeter
+ * @author        Pierre Camaly de Brosses <p.de-brosses2@exeter.ac.uk>
+ * @license       http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+@mixin UI-elements {

```

Comment:
Building the SCSS snippet with a mixin is generally fine, but please use a more unique mixing name with all lowercase characters. The reason is that the SCSS snippet will be added to the full SCSS stack of Boost Union and we want to avoid any name clashes.
Something like `@mixin busnippet-accessiblelinks-uielements` should be fine.